types.ts

```diff
@@ -41,19 +41,19 @@ interface Status {
     baseline_high_date?: string;
     /** Browser versions that most-recently introduced the feature */
     support: {
-        [K in browserIdentifier]?: string;
+        [K in BrowserIdentifier]?: string;
     };
 }
 
-interface SupportStatus extends Status {
+export interface SupportStatus extends Status {
     /** Statuses for each key in the feature's compat_features list, if applicable. Not available to the npm release of web-features. */
     by_compat_key?: Record<string, Status>
 }
 
 /** Specification URL
  * @format uri
 */
-type specification_url = string;
+export type specification_url = string;
```

Comment:
This type only exists so that we get the `uri` format in JSON Schema. In TypeScript, a type alias for `string` seems kinda pointless (especially with an ugly name like this one). Maybe we can leave this out too?